source/core/src/main/com/csse3200/game/components/tasks/BulletStormTask.java

```diff
@@ -64,14 +67,31 @@ public void start() {
 
         // Enable the BossRangeAttackComponent
         BossRangeAttackComponent rangeAttack = owner.getEntity().getComponent(BossRangeAttackComponent.class);
+        int numShot = (int) (Math.random() * 8);
```

Comment:
use ServiceLoactor.getRandom() instead of Math.random

Reply:
Was useless and removed